common/sagas/schedule/currentScheduleTimezone.ts

```diff
@@ -0,0 +1,22 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { SagaIterator } from 'redux-saga';
+import { TypeKeys } from 'actions/schedule/constants';
+import { SetScheduleTimezoneAction, setScheduleTimezone } from 'actions/schedule';
+import { SetCurrentScheduleTimezoneAction } from 'actions/schedule/actionTypes/scheduleTimestamp';
+
+export function* setCurrentScheduleTimezone({
+  payload: raw
+}: SetCurrentScheduleTimezoneAction): SagaIterator {
+  const value = raw;
+
+  yield call(setField, { value, raw });
+}
+
+export function* setField(payload: SetScheduleTimezoneAction['payload']) {
```

Comment:
This function isn't used anywhere externally, would rather just have inline yield put(setScheduleTimezone(payload)); in the two references above to avoid extra indirection